pkg/translator/prometheusremotewrite/metrics_to_prw_v2.go

```diff
@@ -131,9 +141,40 @@ func (c *prometheusConverterV2) addSample(sample *writev2.Sample, lbls []prompb.
 		off = c.symbolTable.Symbolize(l.Value)
 		buf = append(buf, off)
 	}
-	ts := writev2.TimeSeries{
+
+	sig := timeSeriesSignature(lbls)
```

Comment:
I think you should here follow the same logic as in `prometheusConverter.addSample`, and use a method `getOrCreateTimeSeries` to obtain the time series to add the sample to. Creating a new time series for every sample even if the time series already exists is bad for performance.

Reply:
+1

Let's keep the code consistent between v1 and v2.